packages/venia-ui/lib/components/Checkout/checkoutButton.css

```diff
@@ -0,0 +1,4 @@
+.icon {
+    composes: root from '../Icon/icon.css';
+    --stroke: white;
```

Comment:
Do we prefer using variable overrides on root instead of just targeting the icon directly? This seems easier to me, was just curious.

```css
.icon {
    stroke: white;
}
```

Reply:
I was following the pattern set forth by @jimbo in his recent icon styling PR: https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/MiniCart/section.css#L29